compiler/src/iree/compiler/Codegen/LLVMGPU/Utils/LLVMGPULayoutAnalysisAndDistribution.cpp

```diff
@@ -95,58 +95,54 @@ enum class MMAType {
 
 enum class MMAMatrixType { AMatrix, BMatrix, CMatrix };
 
-static std::array<Dimension, 3> getMMADimensions(MMAType mmaType,
-                                                 MMAMatrixType matrixType,
-                                                 int dim) {
+static std::array<Dimension, 3>
+getMMADimensions(MMAType mmaType, MMAMatrixType matrixType, int dim) {
   switch (mmaType) {
-    case MMAType::M16N8K16:
-      switch (matrixType) {
-        case MMAMatrixType::AMatrix:
-          if (dim == 0)
-            return {{{DimType::LaneIdY, 8},
-                     {DimType::VecIdZ, 2},
-                     {DimType::LaneIdZ, 1}}};
-          return {{{DimType::VecIdX, 2},
-                   {DimType::LaneIdX, 4},
-                   {DimType::VecIdY, 2}}};
-        case MMAMatrixType::BMatrix:
-          if (dim == 0)
-            return {{{DimType::LaneIdY, 8},
-                     {DimType::LaneIdZ, 1},
-                     {DimType::VecIdZ, 1}}};
-          return {{{DimType::VecIdX, 2},
-                   {DimType::LaneIdX, 4},
-                   {DimType::VecIdY, 2}}};
-        case MMAMatrixType::CMatrix:
-          if (dim == 0)
-            return {{{DimType::LaneIdY, 8},
-                     {DimType::VecIdY, 2},
-                     {DimType::LaneIdZ, 1}}};
-          return {{{DimType::VecIdX, 2},
-                   {DimType::LaneIdX, 4},
-                   {DimType::VecIdZ, 1}}};
-      }
-      return {};
-    default:
-      return {};
+  case MMAType::M16N8K16:
+    switch (matrixType) {
+    case MMAMatrixType::AMatrix:
+      if (dim == 0)
+        return {{{DimType::LaneIdY, 8},
+                 {DimType::VecIdZ, 2},
+                 {DimType::LaneIdZ, 1}}};
+      return {
+          {{DimType::VecIdX, 2}, {DimType::LaneIdX, 4}, {DimType::VecIdY, 2}}};
```

Comment:
Weird formatting / line breaks here. Could use braces around the conditions and/or comments to force newlines in the returned values.

Reply:
I will fix this in a follow-up PR